src/components/Menu/Menu/Menu.tsx

```diff
@@ -62,6 +94,7 @@ const Menu = forwardRef(
     const children = useMemo(() => {
       const allChildren = React.Children.toArray(originalChildren);
       return allChildren.filter(child => {
+        // @ts-ignore
```

Comment:
Why?

Reply:
This static `type` properties are throwing following error
`TS2339: Property 'type' does not exist on type 'string | number | ReactElement<any, string | JSXElementConstructor<any>> | ReactFragment | ReactPortal'.   Property 'type' does not exist on type 'string'.`
What would be a solution here @sahariko? We have this problem in many places